<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExpressionBuilder</class>
 <widget class="QMainWindow" name="ExpressionBuilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout" rowstretch="3,0" columnstretch="1,0,0">
      <item row="0" column="2">
       <widget class="QGroupBox" name="operatorsGroupBoxGridLayout">
        <property name="title">
         <string>Operators</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="1" column="0">
          <widget class="QListView" name="operatorsListView"/>
         </item>
         <item row="0" column="0">
          <widget class="QComboBox" name="operatorsComboBox">
           <item>
            <property name="text">
             <string>All</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Trigonometric</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hyperbolic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Rounding</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sums, products, differences</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Exponents and logarithms</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Arithmetic</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QGroupBox" name="expressionGrpupBoxGridLayout">
        <property name="title">
         <string>Expression</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0" colspan="3">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QTextEdit" name="expressionTextEdit"/>
           </item>
          </layout>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="applyPushButton">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="cancelPushButton">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Layers</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QGridLayout" name="rasterCollectionGroupBoxGridLayout">
           <item row="0" column="0">
            <widget class="QListView" name="layersListView"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>682</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
